compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1447,6 +1445,12 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
             {
                 self.tables.coroutine_kind.set(def_id.index, Some(coroutine_kind))
             }
+            if let DefKind::Static(_) = def_kind {
+                if !self.tcx.is_foreign_item(def_id) {
+                    let data = self.tcx.eval_static_initializer(def_id).unwrap();
+                    record!(self.tables.eval_static_initializer[def_id] <- data);
+                }
```

Comment:
If you want a review of these changes you'll have to find someone else to do it, I have no clue what is happening here.